builds/linux/almalinux/8/linux-almalinux.pkrvars.hcl.example

```diff
@@ -8,7 +8,7 @@
 
 // Guest Operating System Metadata
 vm_guest_os_name     = "almalinux"
-vm_guest_os_version  = "8.9"
+vm_guest_os_version  = "8.10"
```

Comment:
Did you mean to change this here?

Reply:
i did, i noticed it was missed by me in the past, i can add it to the changelog